src/lib/Grammar.mly

```diff
@@ -147,9 +147,9 @@ plain_atomic_term_except_name:
   | COF
     { Cof }
   | TOPC
```

Comment:
I prefer to treat this equivalence in elaboration, not in parsing. So: 

1. It's fine to remove the two tactics from `Refiner` because they really are redundant
2. For now I'd like to keep the concrete syntax constructors.

Reply:
Okay!